assets/app/features/upgrade/templates/landing-page_step3.partial.jade

```diff
@@ -0,0 +1,12 @@
+.upgrade(ng-class="{'active': upgradeLandingVm.mode.valid}")
+    .step-title.clearfix
+        crowbar-circle.pull-left(ng-class="{'active': upgradeLandingVm.mode.valid}") 3
```

Comment:
I did understand 1/10th of those words.

I will check this further, I dont know a lot about directives : <